tfx/types/standard_component_specs.py

```diff
@@ -351,12 +351,16 @@ class TransformSpec(ComponentSpec):
   INPUTS = {
       'examples': ChannelParameter(type=standard_artifacts.Examples),
       'schema': ChannelParameter(type=standard_artifacts.Schema),
+      'cache_input_path': ChannelParameter(
+          type=standard_artifacts.TransformCache, optional=True),
   }
   OUTPUTS = {
       'transform_graph':
           ChannelParameter(type=standard_artifacts.TransformGraph),
       'transformed_examples':
           ChannelParameter(type=standard_artifacts.Examples),
+      'cache_output_path':
+          ChannelParameter(type=standard_artifacts.TransformCache),
```

Comment:
I thought it is possible now to have optional outputs?
See: https://github.com/tensorflow/tfx/blob/deb618730dc7675c8e9dc75e03b8ca795d49653d/tfx/types/standard_component_specs.py#L361

Reply:
Just saw that PR as well and updated PR too have `cache_output_path` similar to `transformed_examples`.